[Bug]: Missing packages when building termux-tools packages

### Problem description

I am tried to update termux tools for my app but I got error of missing dependency.

### What steps will reproduce the bug?

Build termux-tools package without argument `-I`

### What is the expected behavior?

_No response_

### System information

```shell
Run declare -a packages
  declare -a packages
  for repo_path in $(jq --raw-output 'del(.pkg_format) | keys | .[]' repo.json); do
    repo=$(jq --raw-output '.["'${repo_path}'"].name' repo.json)
    if [ -f ./built_${repo}_packages.txt ]; then
      packages="$packages $(cat ./built_${repo}_packages.txt | tr '\n' ' ')"
    fi
  done
  
  if [ true == 'false' ]; then
    NDK=$ANDROID_NDK ANDROID_HOME=$ANDROID_SDK_ROOT ./build-package.sh -a arm $packages
  elif [ -n "$packages" ]; then
    ./scripts/run-docker.sh ./build-package.sh -a arm $packages
  fi
  shell: /usr/bin/bash -e {0}
Running container 'termux-package-builder' from image 'ghcr.io/termux/package-builder'...
Creating new container...
Unable to find image 'ghcr.io/termux/package-builder:latest' locally
latest: Pulling from termux/package-builder
9b857f539cb1: Pulling fs layer
d24da82b95f3: Pulling fs layer
2c4c2fc3faee: Pulling fs layer
de5c9067ef5f: Pulling fs layer
74dee1923210: Pulling fs layer
b7c9d133a0f7: Pulling fs layer
de5c9067ef5f: Waiting
74dee1923210: Waiting
b7c9d133a0f7: Waiting
d24da82b95f3: Verifying Checksum
d24da82b95f3: Download complete
2c4c2fc3faee: Download complete
de5c9067ef5f: Download complete
9b857f539cb1: Verifying Checksum
9b857f539cb1: Download complete
b7c9d133a0f7: Verifying Checksum
b7c9d133a0f7: Download complete
9b857f539cb1: Pull complete
d24da82b95f3: Pull complete
2c4c2fc3faee: Pull complete
de5c9067ef5f: Pull complete
74dee1923210: Verifying Checksum
74dee1923210: Download complete
74dee1923210: Pull complete
b7c9d133a0f7: Pull complete
Digest: sha256:c6ad5a9da424b11a9b5d25837daf73fac6b66d7824909b1420d49b54951c831d
Status: Downloaded newer image for ghcr.io/termux/package-builder:latest
861706f17f687957ffc39cab697ba94de3a2f694636dacced7cb5b9a2c0e1b59
termux - building termux-tools for arch arm...
Downloading https://github.com/termux/termux-elf-cleaner/releases/download/v2.2.1/termux-elf-cleaner
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

100  471k  100  471k    0     0  2241k      0 --:--:-- --:--:-- --:--:-- 2241k
ERROR: Package libimobiledevice depends on non-existing package "usbmuxd"
ERROR: Obtaining buildorder failed
Error: Process completed with exit code 1.
```

--- a/packages/libimobiledevice/build.sh
+++ b/packages/libimobiledevice/build.sh
@@ -6,11 +6,12 @@ _COMMIT=860ffb707af3af94467d2ece4ad258dda957c6cd
 _COMMIT_DATE=20230430
 TERMUX_PKG_MAINTAINER="@termux"
 TERMUX_PKG_VERSION=1.3.0-p${_COMMIT_DATE}
+TERMUX_PKG_REVISION=1
 TERMUX_PKG_SRCURL=git+https://github.com/libimobiledevice/libimobiledevice
 TERMUX_PKG_SHA256=5b88f3b348f06fe9d1b5ffc639f813b5956d757e527d5c7ca724a64c1b0b3b4e
 TERMUX_PKG_GIT_BRANCH=master
 TERMUX_PKG_AUTO_UPDATE=false
-TERMUX_PKG_DEPENDS="libimobiledevice-glue, libplist, libusbmuxd, openssl, usbmuxd"
+TERMUX_PKG_DEPENDS="libimobiledevice-glue, libplist, libusbmuxd, openssl"
 
 termux_step_post_get_source() {
 	git fetch --unshallow

--- a/packages/libusbmuxd/build.sh
+++ b/packages/libusbmuxd/build.sh
@@ -5,12 +5,12 @@ TERMUX_PKG_MAINTAINER="@termux"
 _COMMIT=f47c36f5bd2a653a3bd7fb1cf1d2c50b0e6193fb
 _COMMIT_DATE=20230430
 TERMUX_PKG_VERSION=2.0.2-p${_COMMIT_DATE}
-TERMUX_PKG_REVISION=1
+TERMUX_PKG_REVISION=2
 TERMUX_PKG_SRCURL=git+https://github.com/libimobiledevice/libusbmuxd
 TERMUX_PKG_SHA256=c415b8859ccae494387c14dc9c297910843b16b5ba62aab15d51116137b513c4
 TERMUX_PKG_GIT_BRANCH=master
 TERMUX_PKG_AUTO_UPDATE=false
-TERMUX_PKG_DEPENDS="libimobiledevice-glue, libplist, libusb, usbmuxd"
+TERMUX_PKG_DEPENDS="libimobiledevice-glue, libplist, libusb"
 TERMUX_PKG_EXTRA_CONFIGURE_ARGS="
 --without-preflight
 --without-systemd

